requirements.txt

```diff
@@ -23,3 +23,4 @@ tqdm
 termcolor
 wandb
 ipykernel
+ipywidgets
```

Comment:
Please update also [LCRS.egg-info/PKG-INFO](https://github.com/DaniAffCH/lcrs/blob/main/LCRS.egg-info/PKG-INFO) and [LCRS.egg-info/requires.txt](https://github.com/DaniAffCH/lcrs/blob/main/LCRS.egg-info/requires.txt)

Reply:
@Jan-Koll reminder